sysl2/Grammar1_test.go

```diff
@@ -6,25 +6,29 @@ import (
     sysl "anz-bank/sysl/sysl2/proto"
 )
 
-func testParser(g *sysl.Grammar, numTerms int, tokens []int, text string, expected bool, t *testing.T) {
+func testParser(g *sysl.Grammar, numTerms int, tokens []int, text string, expectedResult bool, t *testing.T) (bool, []interface{}) {
```

Comment:
How about  something like this (pseudo-code):
```
expected = {
    grammarTermsCnt: 6, 
    grammarParsesText: true, 
    textTokens: [2,3,2,-1]
}
testParser(grammar, text, expected)
```
 or at least `grammar` and `text` as the first two params and all other params prefixed with `expected`?

Reply:
more effort than it is worth at the moment. Will keep this in mind for later...

Reply:
kk